.changeset/seven-zebras-sit.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-core": major
```

Comment:
**note:** Since this changes the behavior of the `useIsMounted` hook, I thought a `major` release was appropriate. I could be convinced that this is `minor` though if folks disagree 😄